packages/redux-routine/src/test/index.js

```diff
@@ -85,6 +87,33 @@ describe( 'createMiddleware', () => {
 		return store.dispatch( createAction() );
 	} );
 
+	/**
```

Comment:
This shouldn't take JSDoc form, since it's not documenting an object or definition.

Reply:
Fix pushed.